src/main/java/org/broadinstitute/hellbender/tools/exome/ReadCountCollection.java

```diff
@@ -104,10 +77,32 @@ public ReadCountCollection(final SetUniqueList<Target> targets, final SetUniqueL
      * @param columnNames column name list, not a {@code null}, does not contain any {@code null}, does not contain repeats.
      * @param counts count matrix, not a {@code null}, has as many rows as {@code targets} elements and as many columns as {@code columnNames} elements.
      */
-    private ReadCountCollection(final List<Target> targets, final List<String> columnNames, final RealMatrix counts) {
-        this.targets = targets;
-        this.columnNames = columnNames;
-        this.counts = counts;
+    private ReadCountCollection(final List<Target> targets, final List<String> columnNames, final RealMatrix counts, final boolean verifyInput) {
```

Comment:
This is the same behavior as before: the private non-verifying constructor is useful when invoked by `subsetTargets`, `subsetColumns`, and `arrangeTargets`.